lua/starfall/libs_cl/render.lua

```diff
@@ -1776,6 +1811,14 @@ function render_library.overrideBlend(on, srcBlend, destBlend, blendFunc, srcBle
 	end
 end
 
+--- Returns the current alpha blending
+--return number Blending in the range 0 to 1
+function render_library.getBlend(alpha)
+	if not renderdata.isRendering then SF.Throw("Not in a rendering hook.", 2) end
+
```

Comment:
Removing this empty line would be more consistent with other functions you're adding.